### PR TITLE
initial pass for azure vm

### PR DESCRIPTION
**Pull request by michaelcramer**

### CODE REVIEW

**Comment by michaelcramer** [comment]
Currently there's are some issues after everything comes up.    The container `retool-onpremise_db-connector` keeps restarting with error `/bin/sh: 1: ./docker-scripts/start_api.sh: not found`

File: modules/azure_vm/README.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+      protocol                   = "Tcp"
+      source_port_range          = "*"
+      destination_port_range     = "3000"
+      source_address_prefix      = "*"
+      destination_address_prefix = "*"
+    }
+  ]
+```
+
+By default, this module creates a publicly-accessible security group that enables inbound traffic on ports (`30`, `443`, `22`, and `3000`) and all outbound traffic.
```
Comment (prashantmital) [inline modules/azure_vm/README.md:130]: Seems like this section describes the settings specified in the preceding code block? If so, can we move it before the code block and rephrase to something like:
```
For example, to create a Retool instance accessible from anywhere, you can use the following value for `security_group` which enables inbound traffic on ports (`30`, `443`, `22`, and `3000`) and all outbound traffic. Note that this is also the default behavior of this module.
```

**Review by prashantmital — CHANGES_REQUESTED** [review]

File: modules/azure_vm/README.md [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+  }
+}
+
+provider "azurerm" {
+  features {}
+}
+
+
+module "retool" {
+  source               = "git@github.com:tryretool/retool-terraform.git//modules/azure_vm"
```
[the remark this answers is not in the document]
Comment (michaelcramer) [reply modules/azure_vm/README.md:27]: yup!

File: modules/azure_vm/README.md [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+```
+docker-ps
+```
+
+9. Modify your environment variables. If you have an external RDS database (strongly recommended), replace the `POSTGRES_` environment variables with the new ones.
+
+- If testing out your instance for the first time without SSL/HTTPS, you should uncomment `# COOKIE_INSECURE = true`
+- Replace your `LICENSE_KEY` with your provided Retool license key
+
+10. Add any additional configuration needed. You can refer to our documentation for [all additional environment variables](https://docs.retool.com/docs/environment-variables).
```
[the remark this answers is not in the document]
Comment (michaelcramer) [reply modules/azure_vm/README.md:72]: they'd be as environment variables, i'm not sure what additional specification would be needed?

File: modules/azure_vm/README.md (hunk 1 above)
Comment (michaelcramer) [reply modules/azure_vm/README.md]: done

File: modules/azure_vm/README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (prashantmital) [reply modules/azure_vm/README.md:72]: As in, could they be added to a specific TF file in this module? Or would they need to be manually defined by someone logged onto the instance?

**Pull request closed without merging**